README.md

```diff
@@ -232,6 +232,7 @@ The client keys used with Parse are no longer necessary with Parse Server. If yo
 #### Advanced options
 
 * `fileKey` - For migrated apps, this is necessary to provide access to files already hosted on Parse.
+* `filesCacheControl` - Set `Cache-Control` header when serving files with builtin files router. Defaults to `public, max-age=86400` (1 day).
```

Comment:
Thanks for adding this. Still not sure about the default value though...@flovilmart do you want to weigh in on this?